data-pipeline/src/trace_exporter.rs

```diff
@@ -153,15 +153,28 @@ impl TraceExporter {
     }
 }
 
-#[derive(Default)]
 pub struct TraceExporterBuilder {
     host: Option<String>,
     port: Option<u16>,
-    tracer_version: Option<String>,
-    language: Option<String>,
-    language_version: Option<String>,
-    interpreter: Option<String>,
-    no_proxy: bool,
+    tracer_version: String,
+    language: String,
+    language_version: String,
+    language_interpreter: String,
+    use_proxy: bool,
+}
+
+impl Default for TraceExporterBuilder {
```

Comment:
I'm split on whether or not this is a good idea. On the one hand, we should avoid negative bool variables like `no_proxy` when possible as it can easily lead to confusion versus bools that are affirmatively named. On the other hand, I see why `no_proxy` was probably chosen in the first place since we want the default behavior to use the proxy and bools default to false naturally. Instead of using derive Default macro we had to implement it manually which can seem like overkill for just one struct field.

I can see an argument that this change makes the code more complicated in its attempt to make it simpler so I can revert back to `no_proxy` if anyone feels strongly.